packages/github/src/github.ts

```diff
@@ -14,7 +14,12 @@ export const context = new Context.Context()
  */
 export function getOctokit(
   token: string,
-  options?: OctokitOptions
+  options?: OctokitOptions,
+  ...additionalPlugins: OctokitPlugin[]
```

Comment:
Since `options` is optional what happens here if the caller doesn't pass anything in as far as that argument goes? (e.g. `getOctokit(token)`).

Maybe we could set a default value to remedy and make `additionalPlugins` optional with no default as the new "last arg"?

```ts
options?: OctokitOptions = {},
additionalPlugins?: OctokitPlugin[]
```
(I'm sure the typing isn't right there with `OctokitOptions`, but you get the idea).

Reply:
So reading some [docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters#using_rest_parameters_in_combination_with_ordinary_parameters), the default for the "rest parameters" aka "variadic parameters" is an empty array. Typescript won't actually let you make this `...additionalPlugins?: OctokitPlugin[]`

Reply:
My bad, I misunderstood the usage to be something like

```js
const plugins = [Plugin1, Plugin2];
getOctokit(token, options, plugins);
```

instead of 

```js
getOctokit(token, options, Plugin1, Plugin2);
```

Thanks for clarifying!